azure-pipelines.yml

```diff
@@ -1,5 +1,5 @@
 pool:
-  vmImage: 'Ubuntu-16.04'
+  vmImage: 'ubuntu-18.04'
```

Comment:
I suppose we could go straight to `ubuntu-20.04` to delay the moment we need to change this VM image again, but 18.04 is good enough for now